bdao-constitution.md

```diff
@@ -1,6 +1,6 @@
 # **BanklessDAO Constitution & Community Handbook**
 
-Version 1.0
+Version 1.1.1
```

Comment:
Shouldn't it be 1.0.1?

Reply:
Yea, bDIP-00 was a minor update, so changed to 1.1.0
bDIP-01 is patch, so 1.1.1